### PR TITLE
simplify CI

### PR DESCRIPTION
**Pull request by alexandercbooth**
As noted by @rgbkrk in #2, by switching out our docker image we can simplify our CI setup 🙂

### CODE REVIEW

**Comment by alexandercbooth** [comment]
OK, _finally_ good to go now 👍.  Bed time for me 🛌 💤

**Comment by rgbkrk** [comment]
🎉

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
       - run:
           name: Install nteract desktop
           command: |
+            sudo apt install libgif-dev
```
Comment (rgbkrk) [inline .circleci/config.yml]: Interesting.

File: index.js [hunk 2]
```diff
@@ -31,6 +31,9 @@ app
     // Stop the application
     return app.stop();
   })
+  .then(function() {
+    console.log("Success!");
+  })
```
Comment (rgbkrk) [inline index.js]: smart

File: .circleci/config.yml (hunk 1 above)
Comment (alexandercbooth) [reply .circleci/config.yml]: Yeah without this, we were getting a node-gyp error with canvas. It didn't actually break the build, but you know, can't resist getting rid of errors 😄. [Here](https://circleci.com/gh/nteract/desktop-integration-tests/13) is an example of the log with the error.

**Pull request merged**